packages/pages/src/vite-plugin/build/buildStart/rendering/templateUtils.ts

```diff
@@ -19,9 +20,13 @@ const pathToModule = new Map();
  */
 export const readTemplateModules = async (
   feature: string,
-  manifest: Manifest
+  manifest: Manifest,
+  projectStructure: ProjectStructure
 ): Promise<TemplateModuleInternal<any, any>> => {
-  const path = manifest.bundlePaths[feature].replace("assets", "..");
+  const path = manifest.bundlePaths[feature].replace(
+    projectStructure.config.subfolders.assets,
+    ".."
```

Comment:
now that the assets/template directory is always configurable it won't always be one directory away from it right?

Reply:
This has to do with the plugin files in Deno. We set the sourceFiles in ci.json with the follwing:

```
{
    "root": "dist",
    "pattern": "subdirectory/assets/{server,static,renderer,render}/**/*{.js,.css}"
}
```

I believe the way that it works is that everything copied at the pattern path actually gets copied to the root, and since we are replacing the full path here it is actually relative only one level deep to that root.